### DISCUSSION TITLE
Connect lockers to fabricators

### DISCUSSION THREAD

**Discussion by 1v01** [FakeFishGames/Barotrauma#6075] [Ideas and suggestions]
Connect all lockers that are next to the fabricators to those fabricators. I noticed only a few sub has this.

**Comment by Veritius** [comment]
This is a per-sub thing set in the editor. You can open the editor and link it with `Space+LMB`.
I know many users that think this is really annoying, especially with the occasional jank that it introduces, sometimes making fabricators unusable. There's a discussion about making it an option in Settings, but right now, this isn't a very good idea.

**Reply by 1v01** [reply]
I didn't know it's intentional.
